File: java/class5/BUAdmin/Person.java
public class Person {
    
    // INSTANCE VARIABLES
    public String buid;
    public String lastname;
    public String firstname;
    
    // CONSTRUCTORS
    public Person() {
        this.buid = "";
        this.lastname = "";
        this.firstname = "";
    }
    public Person(String buid, String lastname, String firstname) {
        this.buid = buid;
        this.lastname = lastname;
        this.firstname = firstname;
    }
    
    // METHODS
    public void display() {
        System.out.println("BUID: " + this.buid +
                           "\nLastname: " + this.lastname +
                           "\nFirstname: " + this.firstname + "\n\n");
    }
    
    
    
    
    
}

File: python/class5/bu_admin/person.py
class Person:
    
    # initializer (kinda like a java constructor)
    def __init__(self, buid, lastname, firstname):
        # instance variables go in here
        self.buid = buid
        self.lastname = lastname
        self.firstname = firstname

    # method
    def display(self):
        print "BUID: ", self.buid, "\nLastname: ", self.lastname, "\nFirstname: ", self.firstname, "\n\n"
